.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+For bug reports, please respect the following guidelines and check the boxes accordingly.
+
+For questions of the type: "How can this be done?", please consider posting them on Stackoverflow.
+
+For everything else ignore this template.
+
+## Bug report
+
+- [ ] Start a new R session
```

Comment:
I guess most things are obviously bugs or non-bugs when run in any kind of environment; I guess requiring the user to start a new R session (or maybe not even that) will be good enough for like 99% of all cases.